doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -250,6 +251,28 @@ The following is an example of a record with non-trivial subtyping relation:
    E[Γ] ⊢ \mathsf{packType}@\{i\} =_{βδιζη}
    \mathsf{packType}@\{j\}~\mbox{ whenever }~i ≤ j
 
+Specifying cumulativity
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The variance of the universe parameters for a cumulative inductive may be specified by the user.
+
+For the following type, the first universe has its variance
+automatically inferred (it is irrelevant), the second is required to
+be irrelevant, the third is covariant and the last is invariant. With
+these annotations the third and last universes have less general
+variances than would be inferred.
```

Comment:
I think you should not use "positions" in the following text but rather mention the annotation symbol in the doc.
Eg "a universe `u` has no variance annotation; `*u` requires the variance of u to be irrelevant; `+u` makes u covariant..."

Reply:
It would also be nice to have an example/explanation of what variance means (to me it's a very "niche" concept in CS) and possibly a hint to why one may want/need to specify it.

Reply:
What about these comments?

Reply:
lost in the github "hidden items"

Reply:
Updated to stop using positions